cfgov/housing_counselor/jinja2/housing_counselor/improved_index.html

```diff
@@ -180,42 +177,52 @@ <h2 class="h4">
                             </thead>
                             <tbody>
                                 {% for counselor in api_json.counseling_agencies %}
-                                <tr>
-                                    <td data-label="Agency">
+                                <tr id="hud-result-{{ loop.index }}">
+                                    <td>
```

Comment:
I think the `data-label` needs to stay there so the label will appear when the table columns stack on top of each other on small screens

Reply:
Ahh hah, I was wondering about these. Should have checked in CF!

Reply:
Fixed!